bower.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "html-template",
+  "version": "1.0.0-rc.1",
+  "description": "HTML Template polyfill",
+  "main": "template.js",
+  "authors": [
+    "The Polymer Project Authors (https://polymer.github.io/AUTHORS.txt)"
+  ],
+  "license": "BSD-3-Clause",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/webcomponents/template.git"
+  },
+  "keywords": [
+    "html-template",
+    "htmltemplate",
+    "html-templates",
+    "htmltemplates",
+    "web-components",
+    "webcomponents",
+    "polyfill",
+    "shim"
+  ],
+  "homepage": "https://webcomponents.org/polyfills",
+  "ignore": [
+    "**/.*",
```

Comment:
It's needed for excluding files like `.travis.yml`, `.gitignore`, etc. from Bower package.